### PR TITLE
added JDBC batch processing to Employee Registration

### PR DESCRIPTION
**Pull request by hninpwintaung-hpa**
To enable the insertion of multiple employees at once, 
I've implemented a batch insert function and would like to merge it into the 'main' branch. 
Could you please review it? Thank you!

### CODE REVIEW

File: src/main/java/com/employee/controller/EmployeeController.java [hunk 1]
```diff
@@ -11,134 +12,200 @@
...
-		
-		//employeeController.updateEmployee();
-		employeeController.deleteEmployee();
+		employeeController.addEmployeeBatch();
+	}
+
+	public void addEmployeeBatch() {
+		List<Employee> employeesToInsert = new ArrayList<>();
+
+		Employee Alice = new Employee();
```
Comment (YeWin) [inline src/main/java/com/employee/controller/EmployeeController.java]: @hninpwintaung-hpa object name should start with small letter. Not Alice, it's alice.

File: src/main/java/com/employee/controller/EmployeeController.java [hunk 2]
```diff
@@ -11,134 +12,200 @@
...
+		Alice.setEmployeeId(3);
+		Alice.setFirstName("Alice");
+		Alice.setLastName("Johnson");
+		Alice.setEmail("alice@gmail.com");
+		Alice.setHireDate("2023-06-15");
+
+		Employee Eve = new Employee();
```
Comment (YeWin) [inline src/main/java/com/employee/controller/EmployeeController.java]: Same as above review. @hninpwintaung-hpa

**Review by YeWin — CHANGES_REQUESTED** [review]
Please fix the code comments above.
@hninpwintaung-hpa

**Comment by hninpwintaung-hpa** [comment]
@YeWin 
Thank you for reviewing and I've made the changes you suggested. Could you please check it again when you have time?

**Review by YeWin — APPROVED** [review]
Thanks for fixed. It's OK now. @hninpwintaung-hpa

**Pull request merged**